build.py

```diff
@@ -570,6 +570,8 @@ def create_dockerfile_buildbase(ddir, dockerfile_name, argmap, backends):
             unzip \
             wget \
             zlib1g-dev \
+            libc-ares-dev \
```

Comment:
We build c-ares in cmake and we explicitly use the built version there... so why do we need it here?

Reply:
I was trying to apt-get install this instead of adding it as an External Project (because thats how google-cloud-cpp does it) removing it since I reverted back to the external project build in cmake.